src/wcstringutil.cpp

```diff
@@ -256,6 +256,20 @@ wcstring_list_t split_string(const wcstring &val, wchar_t sep) {
     return out;
 }
 
+std::vector<std::string> split_string(const std::string &val, char sep) {
+    std::vector<std::string> out;
+    size_t pos = 0, end = val.size();
+    while (pos <= end) {
+        size_t next_pos = val.find(sep, pos);
+        if (next_pos == wcstring::npos) {
```

Comment:
This needs to be `std::string::npos`. I think they are typically the same value, but I'm fairly certain they don't have to be.